docs-public/snippets.js

```diff
@@ -1005,15 +1007,27 @@ function buildExampleEditorForm(exampleTemplate) {
     return form;
 }
 
-// function getExampleDependencies(directives, production) {
-//     return directives.filter(function(dir) {
-//         return dir.module.indexOf('@progress') === 0 ||
-//                dir.module.indexOf('@telerik') === 0;
-//     }).reduce(function(result, dir) {
-//         result[dir.module] = production ? 'latest' : 'dev';
-//         return result;
-//     }, {});
-// }
+function getExampleDependencies(files, channel) {
+    var imports = {};
+    var dependencyMatcher = /import.*from\s*['|"](.*)['|"]/g;
+    for (var key in files) {
+        if (files.hasOwnProperty(key)) {
```

Comment:
Use `Object.keys` instead of `for..in / hasOwnProperty`. It's shorter, and you can use array methods over it, to get the result.